tasks/update-granules-cmr-metadata-file-links/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@cumulus/update-granules-cmr-metadata-file-links",
+  "version": "2.0.1",
```

Comment:
guessing this version number needs updating now?

Reply:
yeah, good call out - I've updated